calcom/cal.com
# fix: Creating teams for organizations in wizard

## What does this PR do?

When added the possibility to register an org checking if billing was enabled, we introduced a ramification where `requestedSlug` metadata property was not set but we relied on it to let the user create new teams. Expanding that check to cover the other scenario as well.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Go to the org wizard and create teams, it should let you do it in a non-team-billing environment scenario.

## Mandatory Tasks
